app/src/test/java/uk/ac/ebi/atlas/experimentpage/cellplot/JsonCellPlotControllerWIT.java

```diff
@@ -1,109 +1,111 @@
-//                new ClassPathResource("fixtures/experiment-delete.sql"));
-//        populator.execute(dataSource);
-//    }
-//
-//    @BeforeEach
-//    void setUp() {
-//        var experimentAccession = jdbcUtils.fetchRandomPublicExperimentAccession();
-//        jdbcUtils.updatePublicExperimentAccessionToPrivate(experimentAccession);
-//        accessKey = jdbcUtils.fetchExperimentAccessKey(experimentAccession);
-//
-//        var k = tSnePlotSettingsService.getExpectedClusters(experimentAccession);
-//        endpoint_url = "/json/cell-plots/" + experimentAccession + "/clusters/k/" + k.get();
-//
-//        this.mockMvc = MockMvcBuilders.webAppContextSetup(this.wac).build();
-//
-//    }
-//
-//    @Test
-//    void returnsPrivateExperimentWithValidAccessKey() throws Exception {
-//        mockMvc.perform(get(endpoint_url + "?accessKey=" + accessKey + "&n_neighbors=5"))
-//                .andExpect(status().isOk())
-//                .andExpect(content().contentType(MediaType.APPLICATION_JSON_UTF8))
-//                .andExpect(jsonPath("$.series").isArray())
-//                .andExpect(jsonPath("$.series").isNotEmpty());
-//    }
-//
-//    @Test
-//    void returnsErrorWithInValidAccessKey() throws Exception {
-//        mockMvc.perform(get(endpoint_url + "?accessKey=foo_bar&n_neighbors=5"))
-//                .andExpect(status().isBadRequest());
-//    }
-//}
\ No newline at end of file
+package uk.ac.ebi.atlas.experimentpage.cellplot;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.io.ClassPathResource;
+import org.springframework.http.MediaType;
+import org.springframework.jdbc.datasource.init.ResourceDatabasePopulator;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.springframework.test.context.web.WebAppConfiguration;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+import org.springframework.web.context.WebApplicationContext;
+import uk.ac.ebi.atlas.configuration.TestConfig;
+import uk.ac.ebi.atlas.experimentpage.tsneplot.TSnePlotSettingsService;
+import uk.ac.ebi.atlas.testutils.JdbcUtils;
+
+import javax.inject.Inject;
+import javax.sql.DataSource;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@Disabled
```

Comment:
Looks good now instead of commenting the whole class. Could you please provide the reason for disabling this class please? Useful to fellow developers like me :).

Reply:
All the disabled tests are because we had failures in the previous Kubernetes. I added a task in our project to review `@Disabled` tests.